### PR TITLE
Save parameters of all agents through calling world.save()

### PR DESCRIPTION
**Pull request by qipeng**

### CODE REVIEW

**Comment by alexholdenmiller** [comment]
thanks, I'll check it out soon!

File: parlai/core/dict.py [hunk 1]
```diff
@@ -294,6 +294,9 @@ def save(self, filename, append=False, sort=True):
         print('Dictionary: saving dictionary to {}.'.format(filename))
         if sort:
             self.sort()
+
+        filename = self.opt['model_file'] if filename is None else filename
```
Comment (alexholdenmiller) [inline parlai/core/dict.py]: can you move this above the print?

Comment (qipeng) [inline parlai/core/dict.py]: Done!

File: parlai/core/worlds.py [hunk 2]
```diff
@@ -206,6 +206,12 @@ def reset_metrics(self):
         for a in self.agents:
             a.reset_metrics()
 
+    def save(self):
+        """Saves all of the agents in the world by calling their respective
+        save() methods"""
```
Comment (alexholdenmiller) [inline parlai/core/worlds.py]: Nit:
```
"""Saves all of the agents in the world by calling their respective
save() methods.
"""

**Pull request merged**